src/olympia/versions/models.py

```diff
@@ -576,10 +575,6 @@ def from_upload(
         ):
             ScannerResult.objects.filter(upload_id=upload.id).update(version=version)
 
-        if waffle.switch_is_active('enable-uploads-commit-to-git-storage'):
```

Comment:
I'd like to add a migration to delete this switch as well as `git-extraction-cron`. WDYT?

Reply:
Yeah, we can do that with the follow-up migrations that we need to do

Reply:
https://github.com/mozilla/addons/issues/15187